### ISSUE TITLE
Provide a regional file with Canada and UK data

### ISSUE THREAD

**Issue by kinabalu** [pr3d4t0r/COVIDvu#145]

**Comment by pr3d4t0r** [comment]
Do you mean a file akin to the US file, using the BNO News column?  I think so.

I'll work on this after implementing the EU vs Americas regions.

Thanks!

**Comment by kinabalu** [comment]
Yes, I think that would be a great addition after the EU vs Americas regions

**Comment by pr3d4t0r** [comment]
Do we really want this...?  `Пявка` and `vudpatch` could work for this.
